main.py

```diff
@@ -146,6 +148,7 @@ def initialize_game(self):
 
 @dataclass
 class PygameVersion:
+    """ GUI vertion of the game, using pygame-ce """
```

Comment:
If you have a better naming convention for the different games, you can change it.